STACpopulator/cli.py

```diff
@@ -47,15 +48,37 @@ def add_parser_args(parser: argparse.ArgumentParser) -> None:
         implementation_parser = populators_subparser.add_parser(implementation_module_name)
         module.add_parser_args(implementation_parser)
         implementation_parser.add_argument(
-            "--update-collection",
+            "--update-collection-mode",
+            dest="update_collection",
             choices=get_args(inspect.signature(STACpopulatorBase.__init__).parameters["update_collection"].annotation),
```

Comment:
Could this be updated using `UpdateModes` directly (also including `"none"` in it)?  
It would be easier to interpret and there wouldn't be partial subset of possibles values across the code.